packages/react-native-aztec/android/build.gradle

```diff
@@ -12,10 +12,16 @@ buildscript {
         wordpressUtilsVersion = '1.22'
         espressoVersion = '3.0.1'
         aztecVersion = 'v1.3.45'
-        willPublishReactNativeAztecBinary = properties["publishReactNativeAztecVersion"]?.trim() as boolean
+        willPublishReactNativeAztecBinary = properties["willPublishReactNativeAztecBinary"]?.toBoolean() ?: false
```

Comment:
I am still not particularly happy about this naming (and the similar one in `react-native-bridge`), but I can't think of a better name that communicates what it's about.